robolectric/src/main/java/org/robolectric/internal/bytecode/InstrumentationConfiguration.java

```diff
@@ -187,22 +209,13 @@ public boolean shouldAcquire(String name) {
     // See https://github.com/robolectric/robolectric/issues/521
     if (name.equals("android.R$styleable")) return true;
 
-    return !(
-        name.matches(".*\\.R(|\\$[a-z]+)$")
-            || classesToNotAquire.contains(name)
-            || name.startsWith("java.")
-            || name.startsWith("javax.")
-            || name.startsWith("sun.")
-            || name.startsWith("com.sun.")
-            || name.startsWith("org.w3c.")
-            || name.startsWith("org.xml.")
-            || name.startsWith("org.junit")
-            || name.startsWith("org.hamcrest")
-            || name.startsWith("org.specs2")  // allows for android projects with mixed scala\java tests to be
-            || name.startsWith("scala.")      //  run with Maven Surefire (see the RoboSpecs project on github)
-            || name.startsWith("kotlin.")
-            || name.startsWith("com.almworks.sqlite4java") // Fix #958: SQLite native library must be loaded once.
-    );
+    if (packagesToNotAquire.contains(pkgName)) return false;
```

Comment:
Checking for packages preserves previous behavior in most cases (where package name was specified with dot, eg: `"java."`). I guess that omission of dot in remaining package names was unintended mistake.
Also this part of code was private, and now we are adding public api. And using `startsWith` will result in unexpected behavior for clients. `doNotAcquirePackage` suggest that we are excluding this specific package and not all packages with similar name.

With all that said, I don't mind changing it. Will update PR in couple minutes. Thanks for your input.